langchain4j-core/src/main/java/dev/langchain4j/internal/Exceptions.java

```diff
@@ -1,6 +1,7 @@
 package dev.langchain4j.internal;
 
 import dev.langchain4j.Internal;
+import org.jspecify.annotations.Nullable;
```

Comment:
I know that Intellij (annoyingly) insert these kind of annotations when refactoring code, but please let's avoid them.